adhocracy4/modules/models.py

```diff
@@ -95,7 +93,7 @@ class Module(models.Model):
 
     objects = ModulesQuerySet.as_manager()
 
-    blueprint_type = BlueprintTypeField(
+    blueprint_type = models.CharField(
         max_length=255,
         blank=True,
     )
```

Comment:
then we delete the fields with the custom BlueprintType?
Also the new migration cancels the 0008. If we remove 0008, it won't generate 0009 here
@goapunk